jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -16464,7 +16464,7 @@ $as_echo "no (unsupported platform)" >&6; }
 $as_echo "no (explicitly disabled)" >&6; }
   elif test "x$enable_jitserver" = x ; then
     case "$OPENJ9_PLATFORM_CODE" in
-      xa64|xl64|xz64)
+      xa64|xl64|xr64|xz64)
```

Comment:
This file is missing changes from upstream. Please run `bash common/autoconf/autogen.sh` to regenerate it (and `common/autoconf/generated-configure.sh`).

Reply:
I updated the commit after running `bash common/autoconf/autogen.sh`, but I am confused by the huge diffs for the two `generated-configure.sh` files.

Reply:
Please use version 2.69 of autoconf, as was used previously; it seems you used version 2.71.

Reply:
I updated the commit by re-running autogen.sh with autoconf 2.69.